ui/app/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -19,19 +19,22 @@ export default class SignatureRequestOriginal extends Component {
   }
 
   static propTypes = {
-    balance: PropTypes.string,
+    fromAccount: PropTypes.shape({
+      address: PropTypes.string.isRequired,
+      balance: PropTypes.string.isRequired,
```

Comment:
I'm not _entirely_ sure `balance` is always present? 🤔 
Maybe it defaults to zero or something though

Reply:
I think this balance comes from the account tracker? In which case it does default to `undefined` unfortunately - at least until a new block comes in to grab the balances from. It might even stay `undefined` for an extended period of time if using MetaMask offline, so probably best to leave this as optional for now

Reply:
It's not always present, I removed `.required`.